MapleServer2/Types/Status.cs

```diff
@@ -0,0 +1,61 @@
+using MapleServer2.Tools;
+using System;
+
+namespace MapleServer2.Types
+{
+    public class Status
+    {
+        public int UniqueId { get; set; }
+        public int SkillId { get; set; }
+        public int Owner { get; set; }
+        public int Source { get; set; }
+        public int Duration { get; set; }
+        public int Stacks { get; set; }
+        public short Level { get; set; }
+        public int Start { get; set; }
+        public int End { get; set; }
+
+        public SkillCast SkillCast { get; set; }
+
+        public Status() { }
+
+        public Status(SkillCast skillCast, int owner, int source, int duration, int stacks)
+        {
+            SkillId = skillCast.SkillId;
+            UniqueId = GuidGenerator.Int();
+            Owner = owner;
+            Source = source;
+            Duration = duration;
+            Level = skillCast.SkillLevel;
+            Stacks = stacks;
+            SkillCast = skillCast;
+            Start = (int) DateTimeOffset.UtcNow.ToUnixTimeSeconds() + Environment.TickCount;
+            End = Start + 30000;
```

Comment:
What does 30,000 mean? perhaps turn this into a variable or leave a comment documenting it

Reply:
Yeah, the magic number is instead the duration in miliseconds that is already set in the `xml/additionaleffects`, it was set on testing, perhaps they should be added on skillsmetadata.